src/SerialLoops/Controls/ItemListPanel.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Eto.Drawing;
+using Eto.Forms;
+using HaruhiChokuretsuLib.Util;
+using SerialLoops.Lib;
+using SerialLoops.Lib.Items;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SerialLoops.Controls
+{
+    public abstract class ItemListPanel : Scrollable
+    {
+        public List<ItemDescription> Items
+        {
+            protected get { return _items; }
+            set
+            {
+                _items = value;
+                Viewer?.SetContents(GetSections(), _expandItems);
+            }
+        }
+        public SectionListTreeGridView Viewer { get; private set; }
+        
+        protected ILogger _log;
+        private readonly Size _size;
+        private List<ItemDescription> _items;
+        private bool _expandItems { get; set; }
```

Comment:
Another oops from me
```suggestion
        private bool _expandItems;
```